github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```
@@ -1293,3 +1293,60 @@ periodics:
       - name: pgp-bot-key
         secret:
           secretName: pgp-bot-key
+- name: periodic-kubevirt-e2e-k8s-1.21-sig-performance
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cron: "40 7,15,23 * * *"
+  decorate: true
+  cluster: prow-workloads
+  decoration_config:
+    timeout: 4h
+    grace_period: 5m
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
```

This preset only makes sense for windows or rhel tests. It would be interesting to add the presets related to bazel cache instead to speed up the build:
```suggestion
    preset-bazel-cache: "true"
    preset-bazel-unnested: "true"
```

---

Makes sense, thanks!